app-nexus/app-nexus-htb.js

```diff
@@ -0,0 +1,441 @@
+                }
+
+                if (targetingCpm !== '') {
+                    returnParcel.targeting[__baseClass._configs.targetingKeys.om] = [sizeKey + '_' + targetingCpm];
+                }
+
+                returnParcel.targeting[__baseClass._configs.targetingKeys.id] = [returnParcel.requestId];
+                //? }
+
+                //? if(FEATURES.RETURN_CREATIVE) {
+                returnParcel.adm = bidCreative;
+                //? }
+
+                //? if(FEATURES.RETURN_PRICE) {
+                returnParcel.price = Number(__baseClass._bidTransformers.price.apply(bidPrice));
+                //? }
+
+                var pubKitAdId = RenderService.registerAd({
+                    sessionId: sessionId,
+                    partnerId: __profile.partnerId,
+                    adm: bidCreative,
+                    requestId: returnParcel.requestId,
+                    size: returnParcel.size,
+                    price: targetingCpm,
+                    dealId: bidDealId,
+                    timeOfExpiry: __profile.features.demandExpiry.enabled ? __profile.features.demandExpiry.value
+                        + System.now() : 0
+                });
+
+                //? if(FEATURES.INTERNAL_RENDER) {
+                returnParcel.targeting.pubKitAdId = pubKitAdId;
+                //? }
+            }
+        }
+
+        if (!bidReceived) {
+            //? if (DEBUG) {
+            Scribe.info(__profile.partnerId + ' returned no demand for placement: '
+                + returnParcel.xSlotRef.placementId);
+            //? }
+            returnParcel.pass = true;
+        }
+
+        if (__profile.enabledAnalytics.requestTime) {
+            var result = 'hs_slot_pass';
+            if (bidReceived) {
+                result = 'hs_slot_bid';
+            }
+            EventsService.emit(result, headerStatsInfo);
+        }
+    }
+
+    /* =====================================
+     * Constructors
+     * ---------------------------------- */
+
+    (function __constructor() {
+        EventsService = SpaceCamp.services.EventsService;
+        RenderService = SpaceCamp.services.RenderService;
+        ComplianceService = SpaceCamp.services.ComplianceService;
+
+        __profile = {
+            partnerId: 'AppNexusHtb',
+            namespace: 'AppNexusHtb',
+            statsId: 'APNX',
+            version: '2.3.0',
+            targetingType: 'slot',
+            enabledAnalytics: {
+                requestTime: true
+            },
+            features: {
+                demandExpiry: {
+                    enabled: false,
+                    value: 0
+                },
+                rateLimiting: {
+                    enabled: false,
+                    value: 0
+                }
+            },
+            targetingKeys: {
+                id: 'ix_apnx_id',
+                om: 'ix_apnx_om',
+                pm: 'ix_apnx_dealid'
```

Comment:
This `pm` value was changed from the original app-nexus-adapter code. Originally, it was set to `ix_apnx_pm`. Any reason for this change? Keep in mind that changing this property will have an effect on lines setup with old pm key.

Reply:
I have pushed a change to revert this key.  It should now be back to `ix_apnx_pm`.